src/main/java/com/scopely/mapper/JsonDynamoMapper.java

```diff
@@ -93,6 +103,9 @@ public JsonDynamoMapper(AmazonDynamoDB amazonDynamoDB, ObjectMapper objectMapper
         DynamoDBVersionAttribute versionAnnotation;
         for (Method method : methods) {
             if ((versionAnnotation = method.getAnnotation(DynamoDBVersionAttribute.class)) != null) {
+                if (versionAnnotation.attributeName().trim().isEmpty()) {
+                    throw new MappingException("Class " + clazz + " missing attributeName for annotation " + DynamoDBVersionAttribute.class);
```

Comment:
I was thinking it would be nice if we imputed the actual field name-- I imagine that there's a bean standard extractor to use there– but in the meantime we definitely need this check